src/pages/Settings.tsx

```diff
@@ -175,7 +174,7 @@ const Settings: React.FC = () => {
           <SettingsButtonSubmit type="submit">Apply Settings</SettingsButtonSubmit>
         </ButtonsContainer>
       </form>
-      {process.env.NODE_ENV === 'development' && <DevTool control={control} />}
+      {/* process.env.NODE_ENV === 'development' && <DevTool control={control} /> */}
```

Comment:
No longer works?

Reply:
was broken before - also i think v6 auto removes code from `production` - not sure if treeshaking is applied library side tho